lib/health_check/custom_health_checks.rb

```diff
@@ -0,0 +1,50 @@
+module HealthCheck
+  class CustomHealthChecks
+
+    def self.check_sidekiq_redis
+      Sidekiq.redis { |r| "" if r.ping == "PONG" }
+    rescue => e
+       create_error 'sidekiq-redis', e.message
+    end
+
+    def self.check_redis
+      "" if Redis.new.ping
+    rescue Exception => e
+       create_error 'redis', e.message
+    end
+
+    def self.check_s3
+      s3Client = s3_client
```

Comment:
I would have a separate method `s3_client` that returns just the S3 client and memorizes the object like it's done here: https://github.com/AirHelp/ah/blob/master/lib/ah/gateways/s3.rb

Then you won't need to use a variable with a weird name like `s3Client` but will be able to directly call `s3_client` each time.